layers/roads.yaml

```diff
@@ -321,9 +323,10 @@ layers:
 
         minor_road:
             filter:
-                - kind: minor_road
-                - not: { kind_detail: service }
-                - $zoom: { min: 8 }
+                all:
```

Comment:
This doesn't need to be nested in `all` either, can just be this -- **assuming the intent is to only apply the layer when all of the conditions are true**:

```
             filter:
                kind: minor_road
                not: { kind_detail: service }
                $zoom: { min: 8 }
```